0-bootstrap/github.tf.example

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+provider "github" {
+  owner = var.gh_repos.owner
+  token = var.gh_token
+}
+
+locals {
+  cicd_project_id = module.gh_cicd.project_id
+
+  gh_config = {
+    "bootstrap" = var.gh_repos.bootstrap,
+    "org"       = var.gh_repos.organization,
+    "env"       = var.gh_repos.environments,
+    "net"       = var.gh_repos.networks,
+    "proj"      = var.gh_repos.projects,
+  }
+
+  sa_mapping = {
+    for k, v in local.gh_config : k => {
+      sa_name   = google_service_account.terraform-env-sa[k].name
+      attribute = "attribute.repository/${var.gh_repos.owner}/${v}"
+    }
+  }
+
+  commom_secrets = {
+    "PROJECT_ID" : module.gh_cicd.project_id,
+    "WIF_PROVIDER_NAME" : module.gh_oidc.provider_name,
+    "TF_BACKEND" : module.seed_bootstrap.gcs_bucket_tfstate,
+  }
+
+  secrets_list = flatten([
+    for k, v in local.gh_config : [
+      for secret, plaintext in local.commom_secrets : {
+        config          = k
+        secret_name     = secret
+        plaintext_value = plaintext
+        repository      = v
+      }
+    ]
+  ])
+
+  sa_secrets = [for k, v in local.gh_config : {
+    config          = k
+    secret_name     = "SERVICE_ACCOUNT_EMAIL"
+    plaintext_value = google_service_account.terraform-env-sa[k].email
+    repository      = v
+    }
+  ]
+
+  gh_secrets = { for v in concat(local.sa_secrets, local.secrets_list) : "${v.config}.${v.secret_name}" => v }
+
+}
+
+module "gh_cicd" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 12.0"
+
+  name              = "${var.project_prefix}-b-cicd-gh"
```

Comment:
nit: just so it's clear this for managing pools and providers
```suggestion
  name              = "${var.project_prefix}-b-cicd-wif-gh"
```

Reply:
suffix updated